universal-login-react/src/ui/TopUp/TopUpWithCrypto.tsx

```diff
@@ -55,7 +50,7 @@ export const TopUpWithCrypto = ({contractAddress, isDeployment, minimalAmount}:
             />
           </div>
           {isDeployment
-            ? <DeploymentWithCryptoInfo minimalAmount = {minimalAmount}/>
+            ? <MinimalAmountInfo minimalAmount={minimalAmount} paymentMethod='crypto' />
             : <TopUpCryptoInfo/>
```

Comment:
This one should be also shown in the deployment?

Reply:
It hasn't been till today so I don't think so